Sources/muterCore/TestReporting/HTML/HTMLReporter.swift

```diff
@@ -1,183 +1,230 @@
+                    .th("Mutation Score")
+                )
+            ),
+            .tbody(
+                .forEach(fileReports.sorted()) { report in
+                    .tr(
+                        .td(.class("left-aligned"), "\(report.fileName)"),
+                        .td(.class("right-aligned"), "\(report.appliedOperators.count)"),
+                        .td(.class("right-aligned"), .style("color: \(report.scoreColor)"), "\(report.mutationScore)")
+                    )
+                }
+            )
+        )
+    }
+    
+    static func appliedOperators(from testReport: MuterTestReport) -> Self {
+        .div(
+            .class("applied-operators"),
+            .div(
+                .class("toggle"),
+                .input(
+                    .id("show-more-mutation-operators-per-file"),
+                    .type(.checkbox),
+                    .attribute(named: "onclick", value: "showHide(this.checked, 'mutation-operators-per-file');")
+                ),
+                .label(.for("show-more-applied-operators"), "Show all")
+            ),
+            .appliedOperatorsTable(from: testReport.fileReports)
+        )
+    }
+    
+    static func appliedOperatorsTable(from fileReports: [MuterTestReport.FileReport]) -> Self {
+        let reports = fileReports.sorted().flatMap { report in
             report.appliedOperators.compactMap { appliedOperator in
-                """
-                <tr>
-                    <td class="left-aligned">\(report.fileName):\(appliedOperator.mutationPoint.position.line)</td>
-                    <td class="left-aligned"><wbr>\(appliedOperator.mutationPoint.mutationOperatorId.friendlyName)<wbr></td>
-                    <td class="mutation-snapshot">\(appliedOperator.diff)</td>
-                    <td>\(appliedOperator.testSuiteOutcome.asIcon)</td>
-                </tr>
-                """
-            }.joined(separator: "\n")
-        }.joined(separator: "\n")
-
-        return """
-        <table id="applied-operators">
-            \(thead)
-            <tbody>
-            \(tbody)
-            </tbody>
-        </table>
-        """
+                (fileName: report.fileName, appliedOperator: appliedOperator)
+            }
+        }
+        
+        return .table(
+            .id("applied-operators"),
+            .thead(
+                .tr(
+                    .th("File"),
+                    .th("Applied Mutation Operator"),
+                    .th("Changes"),
+                    .th("Mutation Test Result")
+                )
+            ),
+            .tbody(
+                .forEach(reports) { report -> Node<HTML.TableContext> in
+                    .tr(
+                        .td(.class("left-aligned"), "\(report.fileName):\(report.appliedOperator.mutationPoint.position.line)"),
+                        .td(.class("left-aligned"), .raw("<wbr>\(report.appliedOperator.mutationPoint.mutationOperatorId.friendlyName)<wbr>")),
```

Comment:
Do the the `<wb>`s at the start and end want to be a `<wb />` instead?

Reply:
nope, they aren't self enclosed

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr

Reply:
Ew! (I didn't know non-self-closing one-off tags like this were a thing. I always thought that was just allowed for compatibility or leniency.)

Reply:
FWIW me neither lol